go.mod

```diff
@@ -56,19 +56,192 @@ require (
 require (
 	github.com/aquilax/truncate v1.0.0
 	github.com/blang/semver/v4 v4.0.0
-	github.com/opencontainers/image-spec v1.0.2 // indirect
 	gopkg.in/inf.v0 v0.9.1
 )
 
+require (
+	cloud.google.com/go v0.97.0 // indirect
```

Comment:
@mritunjaysharma394  can we merge both require into single require ?

Reply:
i guess there are total 3 require, should we merge all three?

Reply:
Or is it like `direct` or `indirect` require dependencies get managed in 2 different require blocks now from go 1.17 onwards ?

Reply:
Not sure but i guess you are right, here's a go.mod for cosign package: https://github.com/sigstore/cosign/blob/main/go.mod

Reply:
>i guess there are total 3 require, should we merge all three?
No, i mean the 2 with require tags with direct deps. But it should be fine as referenced.

Reply:
Oh, sure, @zeborg, can you please look into this, I am travelling  and my laptop’s battery is dead.

thanks a lot @prateekpandey14 and @zeborg